multi-tenant-configuration/config.py

```diff
@@ -0,0 +1,19 @@
+# Configuration
+
+#Set this to your global admin node
+url = "http://tenant1:8080"
+#If you have multiple tenants use something like
+#url_pattern = "https://{}.example.org"
+#otherwise, url_pattern should be the same as the url variable above
+url_pattern = "http://{}:8080"
```

Comment:
So in the example where I have 'http://tenant1:8080' and 'http://tenant2:8080' as tenants, I would still keep the `pattern_url` ? 

Or do you mean that the script would be more general and also applicable to single-tenant-systems if the pattern is {} ? 
But in this case you would just use the global admin node url, right? 
I thought the pattern was exactly for the case, where there are multiple tenants with similar URLs.

Reply:
I was talking about this comment:
> otherwise, url_pattern should be the same as the url variable above
IMO, if you only have one tenant, url_pattern doesn't even need to be defined. You just use url for everything. The script should be able to deal with that.

If you have tenants URLs that match a pattern, define url_pattern. (Maybe rename to tenant_url_pattern to be clear?)
If they don't match a pattern, define something like tenant_urls instead. The script should use whatever's available. (And if a user defines both... idk, pick one or explode with errors. :D)

And in general, the script should treat 'config key undefined' and 'value is blank' the same.

I hope this is clear? If not, just ask. :D

PS: I just realized that tenant_urls might need to be a map, to know which url belongs to which tenant. But like I said, you could ignore this option for now and add it later. It's not really that important...

Reply:
I think I got it :D 
Just for clarification: 

'config key undefined' -> the user commented out, i.e. 
`# tenant_url_pattern = `

 'value is blank' -> the user inserted an empty string, i.e.
`tenant_url_pattern = ""`